back/spec/models/notification_spec.rb

```diff
@@ -87,49 +87,21 @@
     end
 
     describe 'project_phase_started' do
-      it 'makes project_phase_started notifications on phase started' do
-        phase = create(:active_phase)
-        project = phase.project
-        project.visible_to = 'groups'
-        project.groups << create(:group)
-
-        # members
-        user = create(:user, email: 'user1@example.com', manual_groups: [project.groups.first])
-        _admin = create(:admin, email: 'admin@example.com', manual_groups: [project.groups.first])
-        # non-members
-        _other_user = create(:admin)
-        _other_admin = create(:user, email: 'user2@example.com')
-
-        activity = create(:activity, item: phase, action: 'started')
-
-        notifications = Notifications::ProjectPhaseStarted.make_notifications_on(activity)
-        expect(notifications.map(&:recipient_id)).to match_array [user.id]
-      end
+      let(:project) { create(:project_with_current_phase) }
+      let(:activity) { create(:activity, item: project.phases[2], action: 'started') }
+
+      let(:participant) { create(:user) }
+      let(:non_participant) { create(:user) }
+
+      let(:idea) { create(:idea, project_id: project.id, author_id: participant.id) }
```

Comment:
Maybe I should take this opportunity to use let statements in the existing tests in this spec too? Thoughts?

Reply:
I think `let` is more useful for sharing variables between different specs. For one spec, local variables are fine.